template/build/core/phing/tasks/frontend.xml

```diff
@@ -1,53 +1,9 @@
 <project name="frontend" default="frontend:install">
 
-  <target name="frontend:build" depends="frontend:install" description="Uses gulp to build front end dependencies for all themes.">
-    <if>
-      <isset property="project.themes"/>
-      <then>
-        <foreach list="${project.themes}" param="frontend.theme" target="frontend:build:run"/>
-      </then>
-    </if>
+  <target name="frontend:build" description="Uses gulp to build front end dependencies for all themes.">
```

Comment:
The description here feels a bit misleading to me. As a user I might expect there to already be some magic built in to find and compile my sass files with gulp. It's not clear by the description that I would need to define the command in my project.yml file. Unless I'm missing something which very well could be the case...